components/ILIAS/Mail/classes/class.ilMailAttachmentTableGUI.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+/**
+ * @author  Michael Jansen <mjansen@databay.de>
+ * @ingroup ServicesMail
+ */
+class ilMailAttachmentTableGUI extends ilTable2GUI
```

Comment:
This table has been abandoned with ILIAS 10.x, there is already a new "UI Table" base implementation.

See: https://docu.ilias.de/goto_docu_wiki_wpage_8069_1357.html